api_tests/btsieve/test.ts

```diff
@@ -158,7 +153,7 @@ describe("Test btsieve API", () => {
                     })
                     .then(res => {
                         res.should.have.status(201);
-                        location = res.headers.location;
+                        location = res.header.location;
                         location.should.be.a("string");
```

Comment:
🤡Same as above. Should be obsolete. 
```suggestion
                         location.should.not.be.empty;
```